src/OrderPage/index.js

```diff
@@ -0,0 +1,24 @@
+import './styles.scss';
+import {useEffect, useState} from "react";
+import MenuItem from "../MenuItem";
+const OrderPage = () => {
+
+    const [menu, setMenu] = useState([])
+    const getMenu = () => {
+        fetch('http://localhost:8080/restaurants/6')
+            .then(data => data.json())
+            .then(menu => {
+                setMenu(menu.foodItems)
+            })
+    }
+
+    useEffect ( () => {getMenu ()}
```

Comment:
Not sure if this will work but can this be refactored down to remove the fat arrow function and put getMenu inside - e.g useEffect(getMenu, [menu])??